src/sentry/api/urls.py

```diff
@@ -674,6 +675,11 @@
         AuthLoginEndpoint.as_view(),
         name="sentry-api-0-auth-login",
     ),
+    re_path(
+        r"^oauth/userinfo/$",
```

Comment:
This is going to make the path `/auth/oauth/userinfo`

Do we want this to sit simply on `/oauth/userinfo` instead?

Reply:
Oh oof.
and actually this'll be 
`/api/0/auth/oauth/userinfo`

Reply:
moved to top level!